inst/private/store_vars_in_python.m

```diff
@@ -0,0 +1,31 @@
+function store_vars_in_python (varname, L)
+%varname
+%L
+  persistent counter = round(100000*rand())
```

Comment:
I noted this before somewhere but its still here: why we have this rand number?  Can just set to 0?